src/neuroglancer/sliceview/renderlayer.ts

```diff
@@ -13,46 +13,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-// We use the approach described in the following paper to determine the
-// intersection between the
-// viewport plane and a given 3-D chunk inside of a WebGL vertex shader:
-//
-// A Vertex Program for Efficient Box-Plane Intersection
-// Christof Rezk Salama and Adreas Kolb
-// VMV 2005.
-// http://www.cg.informatik.uni-siegen.de/data/Publications/2005/rezksalamaVMV2005.pdf
-
+ 
 import {ChunkState} from 'neuroglancer/chunk_manager/base';
 import {ChunkManager} from 'neuroglancer/chunk_manager/frontend';
 import {RenderLayer as GenericRenderLayer} from 'neuroglancer/layer';
-import {SLICEVIEW_RENDERLAYER_RPC_ID, VolumeChunkSpecification, VolumeSourceOptions} from 'neuroglancer/sliceview/base';
-import {MultiscaleVolumeChunkSource, SliceView, VolumeChunkSource} from 'neuroglancer/sliceview/frontend';
+import {SLICEVIEW_RENDERLAYER_RPC_ID, SliceViewChunkSpecification, SliceViewSourceOptions} from 'neuroglancer/sliceview/base';
+import {SliceView, SliceViewChunkSource, MultiscaleSliceViewChunkSource} from 'neuroglancer/sliceview/frontend';
+import {ShaderBuilder, ShaderProgram} from 'neuroglancer/webgl/shader';
 import {RefCounted} from 'neuroglancer/util/disposable';
 import {BoundingBox, mat4, vec3, vec3Key, vec4} from 'neuroglancer/util/geom';
 import {Buffer} from 'neuroglancer/webgl/buffer';
 import {GL} from 'neuroglancer/webgl/context';
 import {makeWatchableShaderError, WatchableShaderError} from 'neuroglancer/webgl/dynamic_shader';
-import {ShaderBuilder, ShaderProgram} from 'neuroglancer/webgl/shader';
 import {getShaderType} from 'neuroglancer/webgl/shader_lib';
 import {RpcId, SharedObject} from 'neuroglancer/worker_rpc';
 
-const DEBUG_VERTICES = false;
-
-/**
- * Amount by which a computed intersection point may lie outside the [0, 1] range and still be
- * considered valid.  This needs to be non-zero in order to avoid vertex placement artifacts.
- */
-const LAMBDA_EPSILON = 1e-3;
 
-/**
- * If the absolute value of the dot product of a cube edge direction and the viewport plane normal
- * is less than this value, intersections along that cube edge will be exluded.  This needs to be
- * non-zero in order to avoid vertex placement artifacts.
- */
-const ORTHOGONAL_EPSILON = 1e-3;
-
-class SliceViewShaderBuffers extends RefCounted {
+export class SliceViewShaderBuffers extends RefCounted {
```

Comment:
This stuff is specific to sliceview/volume, so it would be better to move it there.